.github/workflows/bbw_build_container.yml

```diff
@@ -13,7 +13,7 @@ on:
 
 jobs:
   build:
-    runs-on: ubuntu-24.04
+    runs-on: ${{ matrix.runner || 'ubuntu-latest' }}
```

Comment:
please use `ubuntu-22.04` or `ubuntu-24.04` (not latest).

Reply:
ubuntu-latest is 22.04 currenty in github action terms: https://docs.github.com/en/actions/using-jobs/choosing-the-runner-for-a-job#choosing-github-hosted-runners

Reply:
We can lock ubuntu-latest to ubuntu-22.04 to prevent any future GitHub updates for -latest runner. 
Let's merge this please when possible, I need to come up to some additions in this yml file for fedora40-valgrind.

Reply:
Yep, that was my point, pining version is more predictable.